plugin/user.c

```diff
@@ -184,10 +352,16 @@ int module_start(SceSize argc, const void *args) {
 int module_stop(SceSize argc, const void *args) {
   LOG("stopping module");
   // free hooks that didn't fail
-  //if (g_hooks[0] >= 0) taiHookRelease(g_hooks[0], g_app_start_hook);
+  if (g_hooks[0] >= 0) taiHookRelease(g_hooks[0], g_sceKernelLoadStartModule_SceSettings_hook);
```

Comment:
Is there a need to hook stopunloadmodule as well? I don't know if SceSettings ever unloads a plugin, but can you check?